app/components/Stock/StockQuote.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Icon, Left, Right, Spinner, Subtitle, Text, Title } from 'native-base';
+import theme from '../../theme';
+import {
+  formatPercent,
+  formatTimeFromNow,
+  isPositiveChange,
+} from '../../utils/helpers';
+
+export function StockQuote(props) {
+  const { updateTimestamp } = props;
+  const quote = useSelector(({ stock }) => stock.quote.data);
+  const {
+    latestPrice,
+    change,
+    changePercent,
+    bidPrice,
+    bidSize,
+    askPrice,
+    askSize,
+  } = quote;
+
+  const handlePosNegStyle = val => {
+    return val < 0 ? styles.negative : styles.positive;
+  };
+
+  const _renderQuoteChangeIcon = isPositiveChange(change)
+    ? 'caretup'
+    : 'caretdown';
+
+  if (!latestPrice)
+    return (
+      <View style={styles.container}>
+        <Spinner style={styles.spinner} />
+      </View>
+    );
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.quoteDataContainer}>
+        <Left style={styles.quoteContainer}>
+          <Title>
+            {latestPrice ?? <Spinner />}{' '}
```

Comment:
Do you mean `||` operator instead of `??`
that is a null coalescing operator in C#